e2e/playwright/tests/command/todo.spec.ts

```diff
@@ -0,0 +1,181 @@
+// ***************************************************************
+import {test, expect} from '@e2e-support/test_fixture';
+import TodoMessage from '../../support/components/todo_message';
+import {messages} from '../../support/constants';
+import {getGithubBotDM, waitForNewMessages} from '../../support/utils';
+import {getBotTagFromPost, getPostAuthor} from '../../support/components/post';
+
+const repoRegex = /https:\/\/github.com\/[\w\-]+\/[\w\-]+/;
+const prRegex = /https:\/\/github.com\/[\w\-]+\/[\w\-]+\/pull\/\d+/;
+const issueRegex = /https:\/\/github.com\/[\w\-]+\/[\w\-]+\/issues\/\d+/;
+
+const completeCommands = [
+    {position: 1, cmd: 'connect'},
+    {position: 2, cmd: 'disconnect'},
+    {position: 3, cmd: 'todo'},
+    {position: 4, cmd: 'subscriptions [command]'},
+    {position: 5, cmd: 'issue [command]'},
+    {position: 6, cmd: 'me'},
+    {position: 7, cmd: 'mute [command]'},
+    {position: 8, cmd: 'settings [setting] [value]'},
+    {position: 9, cmd: 'setup [command]'},
+];
+
+export default {
+    connected: () => {
+        test.describe('/github todo command', () => {
+            test('from connected account', async ({pages, page, pw}) => {
+                const {adminClient, adminUser} = await pw.getAdminClient();
+                const URL = await getGithubBotDM(adminClient, '', adminUser!.id);
+                await page.goto(URL, {waitUntil: 'load'});
+
+                const c = new pages.ChannelsPage(page);
+
+                // # Run todo command
+                await c.postMessage('/github todo');
+                await c.sendMessage();
+
+                // # Wait for new messages to ensure the last post is the one we want
+                await waitForNewMessages(page);
+
+                // # Get last post
+                const post = await c.getLastPost();
+                const postId = await post.getId();
+
+                const todo = new TodoMessage(post.container);
+
+                // * Assert that titles are there for each section
+                // Text are fixed and checked inside todo component handler
+                await expect(todo.getTitle('openpr')).toBeVisible();
+                await expect(todo.getTitle('assignments')).toBeVisible();
+                await expect(todo.getTitle('reviewpr')).toBeVisible();
+                await expect(todo.getTitle('unread')).toBeVisible();
+
+                // * Assert that description are there for each section
+                // TODO: Counters may vary and should be explicitely changed once the test accounts are set
+                await expect(todo.getDesc('openpr')).toHaveText('You have 1 open pull requests:');
+                await expect(todo.getDesc('assignments')).toHaveText('You have 19 assignments:');
+                await expect(todo.getDesc('reviewpr')).toHaveText('You have 1 pull requests awaiting your review:');
+                await expect(todo.getDesc('unread')).toHaveText('You have 1 unread messages:');
```

Comment:
Opportunity for UX improvement

```suggestion
                await expect(todo.getDesc('openpr')).toHaveText('You have 1 open pull request:');
                await expect(todo.getDesc('assignments')).toHaveText('You have 19 assignments:');
                await expect(todo.getDesc('reviewpr')).toHaveText('You have 1 pull request awaiting your review:');
                await expect(todo.getDesc('unread')).toHaveText('You have 1 unread message:');
```

Reply:
created ticket for that https://mattermost.atlassian.net/browse/MM-52416